x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -111,17 +126,48 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
 
   const handleFlyoutClose = useCallback(() => setFlyoutAlertIndex(-1), [setFlyoutAlertIndex]);
 
+  // TODO when every solution is using this table, we will be bale to simplify it by just passing the alert index
+  const handleFlyoutAlert = useCallback(
+    (alert) => {
+      const idx = alerts.findIndex((a) =>
+        (a as any)[ALERT_UUID].includes(alert.fields[ALERT_UUID])
+      );
+      setFlyoutAlertIndex(idx);
+    },
+    [alerts, setFlyoutAlertIndex]
+  );
+
+  const basicRenderCellValue = ({
+    data,
+    columnId,
+  }: {
+    data: Array<{ field: string; value: string[] }>;
+    columnId: string;
+  }) => {
+    // any is required here to improve typescript performance
+    const value = data.find((d) => d.field === columnId)?.value ?? [];
+    return <>{value.length ? value.join() : '--'}</>;
+  };
+
   const handleRenderCellValue = useCallback(
     (improper: EuiDataGridCellValueElementProps) => {
       const rcvProps = improper as EuiDataGridCellValueElementProps & EuiDataGridCellValueProps;
-      const alert = alerts[rcvProps.visibleRowIndex];
-      return props.renderCellValue({
+      const alert = alerts[rcvProps.rowIndex];
```

Comment:
Why aren't you using `visibleRowIndex`?

Reply:
it is creating a bug when you click on the cell expand, I just updated the code